narwhals/_pandas_like/series.py

```diff
@@ -424,6 +426,23 @@ def std(
         ser = self._native_series
         return ser.std(ddof=ddof)
 
+    def skew(self) -> Any:
+        np = get_numpy()
+        pd = get_pandas()
+        ser = self._native_series
+        if hasattr(ser, "skew") and not isinstance(ser.dtype, pd.ArrowDtype):
+            return float(ser.skew())
+        else:
+            values = ser.to_numpy()
+            n = len(values)
+            if n < 3:
+                return float("nan")
```

Comment:
i might have misunderstood, but does this set the skew to nan if there's less than 3 values? looks like polars doesn't do that
```
In [2]: pl.Series([1,2]).skew()
Out[2]: 0.0
```

Reply:
+1 on this

Reply:
You're right, this is not necessary. Skew is not that meaningful for 1-2 elements, but now that I think of it no reason for it to be `nan`. Should be resolved now.